site/plugin.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Using Weave Net via Docker Networking
+layout: default
+---
+
+New in Docker version 1.9 is a plugin mechanism to add different
+network providers.
+
+To install, download the `weave` script as per the [readme][].
+
+The Weave Net plugin runs automatically when you `weave launch`.  You
+must still tell the weave peers to connect to one another, either via
+`weave launch` or `weave connect`.
+
+## Starting a container:
+
+    $ docker run --net=weave -ti ubuntu
+
+## Using WeaveDNS for Service Discovery:
+
+You need to pass the additional arguments `--dns` and `-dns-search`,
+for which we provide a helper in the weave script:
+
+    $ docker run --net=weave -h foo.weave.local $(weave dns-args) -tdi ubuntu
+    $ docker run --net=weave -h bar.weave.local $(weave dns-args) -ti ubuntu
+    # ping foo
+
+For more details see the [Weave Service Discovery documentation][service-discovery].
+
+## Advanced Topics
+
+The Weave Net plugin actually provides *two* network drivers to Docker
+- one named `weavemesh` that can operate without a [cluster
+store](#cluster-store) (like Docker's overlay driver) and one named
+`weave` that can only work with one.
+
+### `weavemesh` driver
+
+* Weave handles all co-ordination between hosts (referred to by Docker as a "local scope" driver)
+* Supports only a single network (we create one named `weave` for you automatically)
+* Uses Weave's partition tolerant IPAM
+
+### `weave` driver
+
+* Requires a cluster store (referred to by Docker as a "global scope" driver)
+* Supports multiple networks (these must be created with `docker network create --driver weave ...`)
+* Used with Docker's cluster store based IPAM
+
+## <a name="cluster-store"></a>Using a Cluster Store
+
+There's no specific documentation from Docker on using a cluster
+store, but the first part of [Getting Started with Docker Multi-host Networking][docker-net]
+should point the way.
+
+## Only one network
+
+Although Docker allows users to create multiple networks, weave only
+supports the creation of one network in "mesh" mode. Even with a
+cluster store, all containers are on the same virtual network, but
+Docker will allocate their addresses on different subnets so they
+cannot talk to each other directly.
+
+As a result, for the moment you can have only one network at a time.
+
```

Comment:
These two sections seem to duplicate (and somewhat elaborate) what's in the driver sections. I suggest merging the former into the latter.

Also, bonus points if you can come up with a better heading than "Advanced Topics".